src/calculateRentalCost.js

```diff
@@ -1,10 +1,29 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
   // write code here
+  const DAY_COST = 40;
+  const MANY_DAYS = 7;
+  const SOME_DAYS = 3;
+  const MANY_DAYS_DISCOUNT = 50;
+  const FEW_DAYS_DISCOUNT = 20;
+
+  const daysCost = DAY_COST * days;
+
+  if (days >= MANY_DAYS) {
+    return daysCost - MANY_DAYS_DISCOUNT;
+  }
+
+  if (days >= SOME_DAYS) {
+    return daysCost - FEW_DAYS_DISCOUNT;
+  }
+
+  return daysCost;
```

Comment:
The function `calculateRentalCost` is well-named and descriptive, adhering to the first point of the checklist. Constants are used instead of literals, which is good as per the second point. The use of `const` over `let` is also correctly followed, as per the third point. However, the function could be optimized by removing the unnecessary `else` as per the fourth and fifth checklist points. After the `if` condition on line 18 that contains a `return` statement, the subsequent `if` condition on line 22 does not need to be wrapped in an `else` block because if the condition on line 18 is true, the function will exit and the `else` will never be executed. This will make the code cleaner and more straightforward.